fixtures/uniffi-fixture-time/tests/bindings/test_chronological.kts

```diff
@@ -38,6 +38,11 @@ try {
 val kotlinBefore = Instant.now()
 val rustNow = now()
 val kotlinAfter = Instant.now()
-
 assert(kotlinBefore.isBefore(rustNow) || kotlinBefore.equals(rustNow))
-assert(kotlinAfter.isAfter(rustNow) || kotlinAfter.equals(rustNow))
\ No newline at end of file
+assert(
+        kotlinAfter.isAfter(rustNow) ||
+        kotlinAfter.equals(rustNow) ||
+        // Unfortunately the JVM clock was be lower resolution than the Rust clock,
```

Comment:
nit: Is this meaning to say `... JVM clock  *is*  a lower resolution then the Rust clock`? If so I think swapping the `was be` for `is a` might offer a tad bit more clarity!

For that actual test, I tried to map out all scenarios here and think this covers the ones we "want to care about"

Reply:
Good catch; I think I meant "JVM clock *may* be a lower resolution" (equivocating because the test worked fine in CI but failed consistently on my local machine).